AGENT_INTEGRATIONS.md

```diff
@@ -1,12 +1,12 @@
-## Datadog Agent version 7.49.1
+## Datadog Agent version 7.50.0
```

Comment:
Note to self: fix this whole file, we should maintain an entry for 7.49.1.
Probably also merits a card

Reply:
`git fetch --all --tags` and try again

Reply:
👍